packages/workspace/src/utils/buildable-libs-utils.ts

```diff
@@ -280,7 +280,7 @@ export function updateBuildableProjectPackageJsonDependencies(
           );
           depVersion = readJsonFile(depPackageJsonPath).version;
 
-          packageJson.dependencies[packageName] = depVersion;
+          packageJson.peerDependencies[packageName] = depVersion;
```

Comment:
I think that Nx libraries should still be considered as dependencies. Maybe you should make the npm section below add to peerDependencies instead.

Reply:
@Cammisuli this PR is based in a comment I did in #2879.
I actually was thinking that both the nx and npm dependencies would end up as peerDependencies.
Having nx dependencies as dependencies can cause dependencies trees like this, which I believe is not a desirable situation
```
root
+-- @my-org/foo@1
+-- @my-org/bar@1
|   +-- @my-org/foo@2
```

@bhuber2010 I see that already a month passed and you did not do any changes. Do you plan to continue working on this? I can take over if you are currently busy with other stuff.

Reply:
O wow, totally missed the notification I guess, sorry guys! And to myself, was wondering about this. And yea I agree on both going to peerDeps. Didn't realize I missed that, will do that right now and update this PR.

Reply:
updated